scripts/migrate-users.py

```diff
@@ -0,0 +1,209 @@
+model.engine.echo = True
+
+
+###############################################################
+# Configure connection to Django database
+###############################################################
+django_config = read_json_or_die(os.path.join(CODALAB_HOME,
+                                              'website-config.json'))
+
+# Use default settings as defined in codalab-worksheets
+if 'database' not in django_config:
+    django_config['database'] = {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'codalab.sqlite3',
+    }
+
+# Create database connections
+if django_config['database']['ENGINE'] == 'django.db.backends.mysql':
+    import MySQLdb
+    from MySQLdb.cursors import DictCursor
+    db_params = {
+        'db': django_config['database']['NAME'],
+        'user': django_config['database']['USER'],
+        'passwd': django_config['database']['PASSWORD'],
+        'cursorclass': DictCursor,
+    }
+    if django_config['database'].get('HOST', None):
+        db_params['host'] = django_config['database']['HOST']
+    if django_config['database'].get('PORT', None):
+        db_params['port'] = django_config['database']['PORT']
+    django_db = MySQLdb.connect(**db_params)
+elif django_config['database']['ENGINE'] == 'django.db.backends.sqlite3':
+    import sqlite3
+
+    # Use basic dict factory, not sqlite3.Row, for mutability
+    def dict_factory(cursor, row):
+        d = {}
+        for idx, col in enumerate(cursor.description):
+            d[col[0]] = row[idx]
+        return d
+
+    # Default location: ../codalab-worksheets/codalab/$NAME
+    sqlite_path = os.path.join(os.path.dirname(os.getcwd()),
+                               'codalab-worksheets',
+                               'codalab',
+                               django_config['database']['NAME'])
+    django_db = sqlite3.connect(sqlite_path)
+    django_db.row_factory = dict_factory
+else:
+    raise UsageError("Invalid database engine %r" %
+                     django_config['database']['ENGINE'])
+
+
+###############################################################
+# Begin Transaction
+###############################################################
+
+with model.engine.begin() as bundle_db, django_db as django_cursor:
+    ###############################################################
+    # Get existing users from both databases
+    ###############################################################
+    # General SQL query should work on both MySQL and sqlite3
+    django_cursor.execute("""
+        SELECT
+        cluser.id as b_user_id,  -- prevent clash with user_id column on update
+        cluser.password,
+        cluser.last_login,
+        cluser.username AS user_name,
+        cluser.first_name,
+        cluser.last_name,
+        cluser.email,
+        cluser.date_joined,
+        cluser.organization_or_affiliation AS affiliation,
+        cluser.is_superuser,
+        cluser.is_active,
+        email.verified AS is_verified
```

Comment:
Change this line to

IF(NOT ISNULL(email.verified), email.verified, 0) AS is_verified